src/lib/ledgers.js

```diff
@@ -138,6 +140,32 @@ class Ledgers {
   registerExternalRequestHandler (requestHandler) {
     this._externalRequestHandler = requestHandler
   }
+
+  /**
+   * @param {IlpAddress} prefix
+   * @param {LedgerPlugin} plugin
+   */
+  addPlugin (prefix, plugin) {
+    if (prefix.slice(-1) !== '.') {
+      throw new Error('prefix must end with "."')
+    }
+    plugin.onAny(this._relayEvent)
```

Comment:
`onAny` is in EventEmitter2 but not in the built-in EventEmitter. Can this be implemented another way?